### ISSUE TITLE
Add leaderf to the plugin list

### ISSUE THREAD

**Issue by jdhao** [neurosnap/neovimcraft#3]
[Leaderf](https://github.com/Yggdroot/LeaderF) is a fuzzy finder plugin designed for Vim and Neovim. It works flawlessly across Windows, macOS and Linux, and it is really fast due to its C implementations.

**Comment by neurosnap** [comment]
Thanks for the submission!  I've been trying to keep `neovimcraft` exclusively neovim plugins, not ones that can be used for both.

**Comment by jdhao** [comment]
I see plugins like vista.vim and wilder.vim are also listed, which can also be used by Vim users.

**Comment by neurosnap** [comment]
Good catch!  I think this brings up an interesting decision my this project: do we want to include any plugin that neovim supports or should neovimcraft be exclusive neovim plugins?  

Since `vimawesome.com` includes all vim/neovim supported plugins, I'm inclined to differentiate this project from that one and make this site neovim exclusive plugins.  If that's the case then I think we should remove all listings that work with both vim and neovim.

What are your thoughts?

**Comment by neurosnap** [comment]
For now I'm going to try to stick to explicit neovim plugins.  If you find plugins in my directory that should not be there, please let me know and I'll remove them.  Thanks!

**Comment by jdhao** [comment]
@neurosnap  Thanks for the clarificatin. Keep up the good work! :)
